modules/test/playwright/tests/commerce/commerce-theme-minium/commerceSearch.spec.ts

```diff
@@ -407,62 +397,50 @@ test('COMMERCE-6321 As a buyer, I want to be able to search an Orders entry usin
 		user.id
 	);
 
+	const channel = (
+		await apiHelpers.headlessCommerceAdminChannel.getChannelsPage(site.name)
+	).items[0];
+
+	const order = await apiHelpers.headlessCommerceAdminOrder.postOrder({
+		accountId: account.id,
+		channelId: channel.id,
+		name: 'order1',
+		orderStatus: '1',
+	});
+
 	await performLogout(page);
 	await performLogin(page, 'demo.unprivileged');
 
 	await page.goto(`/web/${site.name}`);
 
-	await commerceLayoutsPage.pendingOrdersLink.click();
-	await commerceLayoutsPage.addOrderButton.click();
-	await commerceLayoutsPage.catalogLink.click();
-	await page.mouse.move(100, 0);
-
-	const orders = await apiHelpers.headlessCommerceAdminOrder.getOrdersPage();
-
-	apiHelpers.data.push({id: orders.items[0].id, type: 'order'});
-
 	await commerceThemeMiniumCatalogPage.focusGlobalSearchBarInput();
-	await commerceThemeMiniumCatalogPage.search(`${orders.items[0].id}`);
-	await commerceThemeMiniumCatalogPage
-		.globalSearchBarCommerceOrderLink(
-			`${orders.items[0].id}`,
-			`${account.name}`
-		)
-		.waitFor({state: 'visible'});
+	await commerceThemeMiniumCatalogPage.search(`${order.id}`);
 
 	await expect(
 		commerceThemeMiniumCatalogPage.globalSearchBarCommerceOrderLink(
-			`${orders.items[0].id}`,
+			`${order.id}`,
```

Comment:
here and below we were searching for a different thing (orderItemId and now orderId). Is it correct?